zstacklib/zstacklib/utils/lvm.py

```
@@ -2743,3 +2747,220 @@ def check_lvmlockd_log():
+def reset_pv_uuids(vg_uuid):
+    """Reset all PV UUIDs in the VG to avoid collision with the source platform."""
+    pvs_output = bash.bash_o(
+        "timeout -s SIGKILL 10 pvs --noheading --nolocking -t "
+        "-Svg_name=%s -oname" % vg_uuid).strip().splitlines()
+
+    succeeded = []
+    for pv_line in pvs_output:
+        pv_name = pv_line.strip()
+        if not pv_name:
+            continue
+        r, o, e = bash.bash_roe("pvchange --uuid %s" % pv_name)
```

Ruff (0.15.5)

[warning] Unpacked variable `o` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)